File: query_gbp.js
const { createClient } = require('@supabase/supabase-js');
const supabase = createClient(
  'https://dwepduvhzuhzeehbeaaz.supabase.co',
  'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImR3ZXBkdXZoenVoemVlaGJlYWF6Iiwicm9sZSI6InNlcnZpY2Vfcm9sZSIsImlhdCI6MTc3NzMwMDY3NSwiZXhwIjoyMDkyODc2Njc1fQ.4gnT-NbFvQp_8PwkCHqzMvt1KGXwyZXH6kpSqwC70qg'
);

async function run() {
  const { data, error } = await supabase
    .from('signals')
    .select('id, symbol, type, status, entry, created_at, signal_ts')
    .eq('symbol', 'GBPUSD')
    .order('created_at', { ascending: false })
    .limit(5);
  console.log(data);
}
run();

File: query_gbp.py
import urllib.request
import json
import ssl

url = "https://dwepduvhzuhzeehbeaaz.supabase.co/rest/v1/signals?symbol=eq.GBPUSD&order=created_at.desc&limit=5"
headers = {
    "apikey": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImR3ZXBkdXZoenVoemVlaGJlYWF6Iiwicm9sZSI6InNlcnZpY2Vfcm9sZSIsImlhdCI6MTc3NzMwMDY3NSwiZXhwIjoyMDkyODc2Njc1fQ.4gnT-NbFvQp_8PwkCHqzMvt1KGXwyZXH6kpSqwC70qg",
    "Authorization": "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImR3ZXBkdXZoenVoemVlaGJlYWF6Iiwicm9sZSI6InNlcnZpY2Vfcm9sZSIsImlhdCI6MTc3NzMwMDY3NSwiZXhwIjoyMDkyODc2Njc1fQ.4gnT-NbFvQp_8PwkCHqzMvt1KGXwyZXH6kpSqwC70qg",
    "Content-Type": "application/json"
}

req = urllib.request.Request(url, headers=headers)
context = ssl._create_unverified_context()
with urllib.request.urlopen(req, context=context) as response:
    data = json.loads(response.read().decode())
    print(json.dumps(data, indent=2))
